pkg/components/velero/restic/restic.go

```diff
@@ -44,11 +44,7 @@ type BackupStorageLocation struct {
 // NewConfiguration returns the default restic configuration.
 func NewConfiguration() *Configuration {
 	return &Configuration{
-		BackupStorageLocation: &BackupStorageLocation{
```

Comment:
Do we also need a documentation change for this?

Reply:
I don't think so.

Reply:
All those 3 parameters are marked in documentation as `required = false`. I though this is because we were setting a default values for them. So they are actually optional?

Reply:
updated the docs.